SeriesHW.ipynb

```diff
@@ -0,0 +1,446 @@
+{
```

Comment:
<p>I guess you couldn't decide between float and long int ...</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>